src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -115,6 +117,11 @@ private static boolean isCreatingSelf(Request req, Auth0UserProfile profile) {
                     LOG.warn("Could not parse user object from request.", e);
                 }
             }
+        } else if (method.equalsIgnoreCase("GET")) {
+            // If this a GET request, it should be against the verification-email route for /user or /application.
+            // (Email verification does not apply to admins.)
+            return uri.endsWith(API_USER_PATH + VERIFICATION_EMAIL_PATH) ||
+                uri.endsWith(OTP_USER_PATH + VERIFICATION_EMAIL_PATH);
```

Comment:
I think each of these conditional blocks should be its own method:
- `isCreatingSelf(Request req)`
- `isRequestingVerificationEmail(Request req)`